### PR TITLE
Nelson rodriguez

### PR DESCRIPTION
**Pull request by Yugenero**
Made some changes to the visual of the sign-in page, and incorporated jpeg.

### CODE REVIEW

**Comment by allisonfain08** [comment]
looks great!

**Review by ShonjoyBhattacharyya — COMMENTED** [review]
Hold up on this, you're merging into main.

File: rnFirebaseAuth/components/login.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const styles = StyleSheet.create({
     flexDirection: "column",
     justifyContent: "center",
     padding: 35,
-    backgroundColor: '#fff'
+    backgroundColor: '#FAF9F6'
```
Comment (MijaKennedy) [inline rnFirebaseAuth/components/login.js:89]: want to keep coloring consistent with styling recommendations from client when she gives them

**Review by MijaKennedy — COMMENTED** [review]
lgtm! just need updated styling info and hi-rez img of logo from client whenever possible

**Review by olivia-mauger — COMMENTED** [review]
Your changes look good - will need to show our client on 10/7 the styling and logo. 

One bug is the logo is so big that we lose the ability to click some of the buttons at the bottom of the screen.

**Review by allisonfain08 — COMMENTED** [review]
looks really good!

**Review by sreya101 — APPROVED** [review]
Looks good to be merged

**Pull request merged**